### PR TITLE
ss/DCOS-44616 Adding Advanced Azure page to install docs.

### PR DESCRIPTION
**Pull request by SarahStegall**
## Description
https://jira.mesosphere.com/browse/DCOS-44616
The Universal Installer team would like to add the following pages to our documentation based on feedback from customers and field:
## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

Added page, added contents as noted.

### CODE REVIEW

**Comment by ck4adventure** [comment]
@yishan-lin Could you please re-check the table to make sure everything is as it should be?

**Comment by ck4adventure** [comment]
Putting up content as I got the correct info from the original page listed in the ticket

**Comment by yishan-lin** [comment]
Hi @ck4adventure as per Slack, let me sit down with you.  I'm not sure you understood the requirements.  These are _still_ the AWS variables.

File: pages/1.12/installing/evaluation/mesosphere-supported-methods/azure/advanced-azure/index.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+  source  = "dcos-terraform/dcos/azurerm"
+  version = "~> 0.1"
+
+  cluster_name = "mydcoscluster"
+  ssh_public_key_file = "~/.ssh/id_rsa.pub"
+  admin_ips = ['198.51.100.0/24']
+
+  num_masters = "3"
+  num_private_agents = "2"
+  num_public_agents = "1"
```
[the remark this answers is not in the document]
Comment (ck4adventure) [reply pages/1.12/installing/evaluation/mesosphere-supported-methods/azure/advanced-azure/index.md]: is that an ok place for that line?

[the remark this answers is not in the document]
Comment (yishan-lin) [reply pages/1.12/installing/evaluation/mesosphere-supported-methods/azure/advanced-azure/index.md]: 👍

**Pull request merged**